r2/r2/lib/eventcollector.py

```diff
@@ -469,7 +475,7 @@ def get_context_data(self, request, context):
 
         data["domain"] = request.host
         data["user_agent"] = request.user_agent
-
+        data["referer"] = request.headers.get("Referer", None)
```

Comment:
I'm going to have to copy paste that.  I'm not sure my fingers are capable of adding an extra "r" at this point.  

(Will do!)